app/src/main/java/com/kickstarter/viewmodels/ProjectViewModel.kt

```diff
@@ -55,13 +55,18 @@ interface ProjectViewModel {
     }
 
     interface Outputs {
+        fun isHorizontalRewardsEnabled(): Observable<Boolean>
```

Comment:
same here. instead of emitting this value, we should have an output that describes the desired outcome. what's supposed to happen in the view if this is true or false? it's not clear. it's also missing documentation